firebase-appdistribution/src/test/java/com/google/firebase/appdistribution/impl/FirebaseAppDistributionServiceImplTest.java

```diff
@@ -506,12 +511,33 @@ private AlertDialog assertAlertDialogShown() {
   }
 
   @Test
-  public void signOutTester_setsSignInStatusFalse() throws InterruptedException {
+  public void signOutTester_setsSignInStatusFalse() {
     firebaseAppDistribution.signOutTester();
-    awaitTermination(lightweightExecutor);
+
     verify(mockSignInStorage).setSignInStatus(false);
   }
 
+  @Test
+  public void signOutTester_unsetsCachedNewRelease()
+      throws InterruptedException, FirebaseAppDistributionException, ExecutionException {
+    Task<AppDistributionReleaseInternal> setCachedNewReleaseTask =
+        firebaseAppDistribution.getCachedNewRelease().set(TEST_RELEASE_NEWER_AAB_INTERNAL);
+
+    // Confirm that the cached new release is initially set
```

Comment:
I'm not a big fan of this pattern - I think, in general, we should trust (but not verify) test setup.
In this case I think you should be able to drop this comment, the blank line above it, and the assert on line 528

Reply:
Sounds good. Removed.